code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -790,3 +790,148 @@ Basically, we fill the time between now and 2s from now with hands based off the
+	reagent_state = LIQUID
+	color = "#520c23"
+	taste_description = "sweat"
+	metabolization_rate = 0.4 * REM
+	overdose_threshold = 25
+	ph = 12.2
+	tox_damage = 0
+
+/datum/reagent/inverse/oxandrolone/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+	var/high_message = pick("You feel unstoppable.", "Giving it EVERYTHING!!", "You feel ready for anything.", "You feel like doing a thousand jumping jacks!")
+	if(SPT_PROB(2, seconds_per_tick))
+		to_chat(affected_mob, span_notice("[high_message]"))
+
+/datum/reagent/inverse/oxandrolone/overdose_process(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+	if(SPT_PROB(25, seconds_per_tick))
+		affected_mob.adjust_bodytemperature(30 * TEMPERATURE_DAMAGE_COEFFICIENT * REM * seconds_per_tick)
+		affected_mob.set_jitter_if_lower(3 SECONDS)
+		affected_mob.adjustStaminaLoss(5 * REM * seconds_per_tick)
+	else if(SPT_PROB(5, seconds_per_tick))
+		affected_mob.vomit(VOMIT_CATEGORY_BLOOD, lost_nutrition = 0, distance = 3)
+		//affected_mob.adjustBruteLoss(10 * REM * seconds_per_tick, updating_health = FALSE)
+		affected_mob.Paralyze(3 SECONDS)
+
+/datum/reagent/inverse/salbutamol
+	name = "Bamethan"
+	description = "Blood thinner that drastically increases the chance of receiving bleeding wounds."
+	reagent_state = LIQUID
+	color = "#ecd4d6"
+	taste_description = "paint thinner"
+	ph = 4.5
+	metabolization_rate = 0.08 * REM
+	tox_damage = 0
+
+/datum/reagent/inverse/salbutamol/on_mob_metabolize(mob/living/L)
+	. = ..()
+	ADD_TRAIT(L,TRAIT_BAMETHAN_BLEED,type)
+
+/datum/reagent/inverse/salbutamol/on_mob_end_metabolize(mob/living/M)
+	. = ..()
+	REMOVE_TRAIT(M,TRAIT_BAMETHAN_BLEED,type)
+
+/datum/reagent/inverse/pen_acid
+	name = "Pendetide"
+	description = "Purges basic toxin healing medications and increases the severity of radiation poisoning."
+	reagent_state = LIQUID
+	color = "#09ff00"
+	ph = 3.7
+	taste_description = "venom"
+	metabolization_rate = 0.25 * REM
+	tox_damage = 0
+
+/datum/reagent/inverse/pen_acid/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	if(holder.has_reagent(/datum/reagent/medicine/c2/seiver))
+		holder.remove_reagent(/datum/reagent/medicine/c2/seiver, 5 * REM * seconds_per_tick)
+	if(holder.has_reagent(/datum/reagent/medicine/potass_iodide))
+		holder.remove_reagent(/datum/reagent/medicine/potass_iodide, 5 * REM * seconds_per_tick)
+	if(holder.has_reagent(/datum/reagent/medicine/c2/multiver))
+		holder.remove_reagent(/datum/reagent/medicine/c2/multiver, 5 * REM * seconds_per_tick)
+
+	. = ..()
+	if(HAS_TRAIT(affected_mob, TRAIT_IRRADIATED))
+		affected_mob.set_jitter_if_lower(10 SECONDS)
+		affected_mob.adjust_disgust(3 * REM * seconds_per_tick)
+		if(SPT_PROB(2.5, seconds_per_tick))
+			to_chat(affected_mob, span_warning("A horrible ache spreads in your insides!"))
+			affected_mob.adjust_confusion(10 SECONDS)
```

Comment:
Might want to cap this confusion otherwise it has some RNG potential of being incredibly obnoxious